frontend/public/components/modals/column-management-modal.tsx

```diff
@@ -0,0 +1,170 @@
+import * as React from 'react';
+// FIXME upgrading redux types is causing many errors at this time
+// eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+// @ts-ignore
+import { useDispatch } from 'react-redux';
+import * as _ from 'lodash';
+import {
+  Alert,
+  DataList,
+  DataListCheck,
+  DataListItem,
+  DataListItemRow,
+  DataListCell,
+  DataListItemCells,
+} from '@patternfly/react-core';
+
+import { createModalLauncher, ModalTitle, ModalBody, ModalSubmitFooter } from '../factory';
+import { setColumnManagementFilter } from '../../actions/ui';
+
+export const MAX_VIEW_COLS = 9;
+
+const OVERRIDES = ['Name'];
+
+const getDataListRows = (
+  rowData: any[],
+  onChange: (checked: boolean, event: any) => void,
```

Comment:
Better type than `any`?

Reply:
updated all event types to use React.SyntheticEvent